openupgrade_scripts/scripts/point_of_sale/15.0.1.0.1/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,140 @@
+---Models in module 'point_of_sale'---
+obsolete model closing.balance.confirm.wizard [transient]
+new model pos.bill
+new model pos.close.session.wizard [transient]
+# NOTHING TO DO
+
+---Fields in module 'point_of_sale'---
+point_of_sale / account.bank.statement.cashbox / is_a_template (boolean)       : DEL
+point_of_sale / account.bank.statement.cashbox / pos_config_ids (one2many)     : DEL relation: pos.config
+point_of_sale / account.bank.statement.line / pos_statement_id (many2one)   : DEL relation: pos.order
+# NOTHING TO DO
+
+point_of_sale / account.move             / pos_payment_ids (one2many)    : NEW relation: pos.payment
+point_of_sale / pos.payment              / account_move_id (many2one)    : NEW relation: account.move
+# NOTHING TO DO: new relation
+
+point_of_sale / account.payment          / force_outstanding_account_id (many2one): NEW relation: account.account
+point_of_sale / account.payment          / pos_payment_method_id (many2one): NEW relation: pos.payment.method
+# NOTHING TO DO: new relations
+
+point_of_sale / account.payment          / pos_session_id (many2one)     : NEW relation: pos.session
+point_of_sale / pos.session              / bank_payment_ids (one2many)   : NEW relation: account.payment
+# NOTHING TO DO: new relation
+
+point_of_sale / barcode.rule             / type (False)                  : selection_keys is now '['alias', 'cashier', 'client', 'discount', 'expiration_date', 'location', 'location_dest', 'lot', 'package', 'package_type', 'packaging_date', 'price', 'product', 'quantity', 'use_date', 'weight']' ('['alias', 'cashier', 'client', 'discount', 'location', 'lot', 'package', 'price', 'product', 'weight']')
+# NOTHING TO DO: same selection options in this module
```

Comment:
It seams that `expiration_date`, `location_dest`, `package_type`, `packaging_date`, `quantity` and `use_date` are no longer valid value for this field. Do they have been moved to other modules ? Is there a replacement value for them ?

Reply:
Whoups, sorry. I read it in the wrong way. :p